sway/tree/container.c

```diff
@@ -519,6 +521,7 @@ static void render_titlebar_text_texture(struct sway_output *output,
 	cairo_set_font_options(c, fo);
 	get_text_size(c, config->font, &width, NULL, &baseline, scale,
 			config->pango_markup, "%s", text);
+	width = width > MAX_WIN_TITLE_LEN ? MAX_WIN_TITLE_LEN : width;
```

Comment:
`LEN` kind of implies character count, but it's pixels.

Instead of hardcoding a limit I'd rather have it based on `con->current->width`. The problem I've found with that is that it does not take into account resizes. On the other hand, titles longer than like a hundred characters probably aren't of interest anyway.

Reply:
Maybe I'm missing something, but this looks pointless: Since `width` is either the actual width of the text, or capped at an arbitrary constant of 8192, which is almost certainly wider than the region the rendered text would be placed into, any text that is wide enough to get ellipsized would get truncated when rendered anyway. This patch would prevent the crash, but I don't see the ellipses ever being visible?

Reply:
@tirsek Your observation is correct. I was looking for something like `con->current->width` but was unable to find it. Thanks for the hinter @minus7